packages/form-core/src/validate/ValidateMixin.js

```diff
@@ -192,7 +192,17 @@ export const ValidateMixinImplementation = superclass =>
         this.validate({ clearCurrentResult: true });
       }
 
-      if (['touched', 'dirty', 'prefilled', 'submitted', 'hasFeedbackFor'].includes(name)) {
+      if (
```

Comment:
Nice. Didn't know demos were broken actually :)

If I would be nitpicking, I would say the ValidateMixin knows nothing about the InteractionStateMixin.
If all elements would extend for either FIeld or Fieldset, we could place it there later (only Listbox (and thus SelectRich and Combobox do not extend from Field and we might want to if we want to support [formatting](https://github.com/ing-bank/lion/issues/994)).

Reply:
Yeahh for subclassers it would indeed be nicer if ValidateMixin lived more agnostically of interaction states, as someone could use the validation system but have their own interaction states or something to align with their design system / methodology. Definitely a good idea for refactoring that later on.